src/ui/SortByRow/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ReactElement } from 'react';
+import React, { JSXElementConstructor, ReactElement } from 'react';
```

Comment:
Can you tell me more about this JSXElementConstructor? Have not seen it before

Reply:
Am this is my mistake, ReactNode would be enough